### PR TITLE
Document adding cuML and fix the gnn_fraud_detection_pipeline example

### LINKED ISSUE #741: [BUG]: Cannot install any cuml version with morpheus
### Version

22.12

### Which installation method(s) does this occur on?

Docker

### Describe the bug.

Here are steps to reproduce the issue
#### Step 1: Pull the image
`sudo docker pull nvcr.io/nvidia/morpheus/morpheus:22.11-runtime`

#### Step 2: Start the container using following command
`sudo docker run --rm -ti --runtime=nvidia --gpus=all --net=host nvcr.io/nvidia/morpheus/morpheus:22.11-runtime bash`

#### Step 3: Installing cuml-22.08 using this command
`mamba install -c rapidsai -c nvidia -c conda-forge cuml=22.08`
it says there are some packages needed to be downgraded. We accepted those changes
Screenshot 2023-03-04 at 2 26 07 AM

#### Step 4: reverting back to old dask version to make it morpheus aligned
`pip install --no-deps dask[distributed]==2022.11.1`
Screenshot 2023-03-04 at 2 24 56 AM

#### Step 5: Trying to importing cudf
Screenshot 2023-03-04 at 2 23 53 AM

#### More
We have tried similar approach keeping the rapids version alignment with morpheus in our mind, but it fails every time. In the Conda yml for Morpheus and cuml there are some version misalignment we have found. E.g - morpheus uses `cuda-python<=1.7.0` but `cuml` requires `cuda-python>=1.7.1`. 

#### Short term:
It would be great if there are any ways to integrate cuml to existing containers. 

#### Long term:
In long term, we are expecting the entire rapids stack integration inside morpheus. Also there are misalignment of rapids versions with morpheus, (e.g. morpheus-23.01 is still using cudf-22.10). It would be nicer to have latest versions of the rapids stack inside morpheus container. 

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by dagardner-nv**
* Install stellargraph and tensorflow via pip, the versions in conda-forge are tied to cuda-11.3
* Use tensorflow-2.12, as prior versions conflicted with our version of protobuf.
* Explicitly install a compatable version of protobuf from the anaconda channel that is not in the conda-forge channel.
* Work-around the Keras model's `predict` method no longer accepting the return value from `HinSAGENodeGenerator.flow` due to switch to tensorflow-2.12

fixes #741

### CODE REVIEW

**Comment by rjurney** [comment]
@pdmack @dagardner-nv any chance this could bump to cuML 22.12?

**Review by mdemoret-nv — APPROVED** [review]

**Comment by dagardner-nv** [comment]
@rjurney unfortunately we are tied to v22.10 for the 23.03 release.
